API.md

```diff
@@ -216,6 +263,61 @@ The pendingEmailAddress field holds the unverified email address when it is bein
 }
 ```
 
+* `GET /respondents?id=3b136c4b-7a14-4904-9e01-13364dd7b972&id=be3711f4-d6c3-45e4-810e-5589b547cc5d`
```

Comment:
Would it be useful to hint at what ID should be used as a parameter, e.g.:

`GET /respondents?id={party_id}`